_posts/2023-01-23-OpenSearch-Flink-Connector.markdown

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+title:  "Apache Flink® Connector for OpenSearch®"
```

Comment:
@reta for visibility. Please let me know of any questions.

Reply:
@jhmcintyre Following the ASF trademark guide we use to add the ® to the most prominent and first text occurrence of a product [1]

[1] https://www.apache.org/foundation/marks/#principles

Reply:
Thanks @reta. I read the Apache guidelines as referring to using the full name (i.e. rather than just "Flink") on first and most prominent usage, rather that specifying inclusion of trademark symbols, per [2]. That said, for this post, can we agree on including trademark symbols on the first usage in the body copy, and removing from headline and later appearances in body copy that I've flagged?

[2] https://www.apache.org/foundation/marks/guide

Reply:
:+1: